src/Imath/ImathExport.h

```diff
@@ -3,15 +3,58 @@
 // Copyright Contributors to the OpenEXR Project.
 //
 
+// See the document in OpenEXR docs/SymbolVisibility.md for background
```

Comment:
Did you mean to commit docs/SymbolVisibility.md? I don't see the file.

Reply:
it's in the openexr tree

Reply:
It'd be clearer and more convenient to include the url:

https://github.com/AcademySoftwareFoundation/openexr/blob/master/docs/SymbolVisibility.md